stetho/src/main/java/com/facebook/stetho/common/android/ViewUtil.java

```diff
@@ -4,9 +4,14 @@
 
 import android.app.Activity;
 import android.content.Context;
+import android.graphics.PointF;
 import android.view.View;
+import android.view.ViewGroup;
 import android.view.ViewParent;
 
+import com.android.internal.util.Predicate;
```

Comment:
Define your own.  It was added in Java 8, but that's a long ways off on Android.